package.json

```
@@ -111,7 +112,8 @@
         "onView:dockerImages",
         "onView:dockerNetworks",
         "onView:dockerRegistries",
-        "onView:dockerVolumes"
+        "onView:dockerVolumes",
+        "vscode-docker.views.help"
```

```suggestion
        "onCommand:vscode-docker.views.help"
```
I'd also move it up with the other commands